core/ui/TiddlerInfo/Tools.tid

```diff
@@ -4,22 +4,13 @@ caption: {{$:/language/TiddlerInfo/Tools/Caption}}
 
 \define lingo-base() $:/language/TiddlerInfo/
 \define config-title()
-$:/config/ViewToolbarButtons/Visibility/$(listItem)$
+$:/config/ViewToolbarButtons/Visibility/$(currentTiddler)$
 \end
-<$set name="tv-config-toolbar-icons" value="yes">
 
-<$set name="tv-config-toolbar-text" value="yes">
+<$list filter="[all[shadows+tiddlers]tag[$:/tags/ViewToolbar]!has[draft.of]]">
```

Comment:
It looks like you're overwriting the current tiddler with this list widget. I think that will cause problems with toolbar button handlers that reference the current tiddler, won't it?